en/docs/deploy/deployment-best-practices/configuration-best-practices.md

```diff
@@ -0,0 +1,232 @@
+# Configuration Best Practices
+
+This guide covers configuration management strategies across deployment models (VM-based, containerized, Kubernetes) and CI/CD pipelines. Core principle: configuration is injected at runtime, never embedded in artifacts.
+
+## VM-Based Deployments
```

Comment:
Use sentece case, only l1 headings should have title case. Check and fix all the places